src/delimited_to_arrays.h

```diff
@@ -0,0 +1,16 @@
+# ifndef ARRAYKIT_SRC_FILE_PARSING_H_
+# define ARRAYKIT_SRC_FILE_PARSING_H_
```

Comment:
Should we name these `ARRAYKIT_SRC_DELIMITED_TO_ARRAYS_H_` or `ARRAYKIT_SRC_DTA_H_`?